book/webapps/navigation.md

```diff
@@ -1,24 +1,24 @@
 # Navigation
 
-We just saw how to serve one page, but say we are making a website like `package.elm-lang.org`. It has a bunch of pages (e.g. [search](https://package.elm-lang.org/), [README](https://package.elm-lang.org/packages/elm/core/latest/), [docs](https://package.elm-lang.org/packages/elm/core/latest/Maybe)) that all work differently. How does it do that?
+Nous venons de voir comment servir une seule page, mais mettons que nous soyons en train de faire un site comme `package.elm-lang.org`. Il se compose de nombreuses pages (par exemple : [search](https://package.elm-lang.org/), [README](https://package.elm-lang.org/packages/elm/core/latest/), [docs](https://package.elm-lang.org/packages/elm/core/latest/Maybe)) qui fonctionnent toutes différemment. Comment fonctionne-t-il ?
 
 
-## Multiple Pages
+## Plusieurs pages
 
-The simple way would be to serve a bunch of different HTML files. Going to the home page? Load new HTML. Going to `elm/core` docs? Load new HTML. Going to `elm/json` docs? Load new HTML.
+Pour faire simple, on pourrait servir plusieurs fichiers HTML. On va sur la page d'accueil ? On charge un HTML. On va sur la page de documentation de `elm/core` ? On charge un HTML. Sur la doc de `elm/json` ? On charge un HTML.
 
-Until Elm 0.19, that is exactly what the package website did! It works. It is simple. But it has some weaknesses:
+Avant Elm 0.19, c'est exactement ce que faisait le site `package.elm-lang.org`. Ça marche bien, c'est simple, mais ça a quelques inconvénients :
 
-1. **Blank Screens.** The screen goes white every time you load new HTML. Can we do a nice transition instead?
-2. **Redundant Requests.** Each package has a single `docs.json` file, but it gets loaded each time you visit a module like [`String`](https://package.elm-lang.org/packages/elm/core/latest/String) or [`Maybe`](https://package.elm-lang.org/packages/elm/core/latest/Maybe). Can we share the data between pages somehow?
-3. **Redundant Code.** The home page and the docs share a lot of functions, like `Html.text` and `Html.div`. Can this code be shared between pages?
+1. **Des écrans vides.** L'écran se vide et reste blanc le temps de charger le nouvel HTML. Ça serait bien d'avoir des transitions, plutôt !
+2. **Requêtes redondantes.** Chaque paquet dispose d'un unique fichier `docs.json`, mais il est chargé à chaque fois qu'on va sur la page d'un module du paquet, comme [`String`](https://package.elm-lang.org/packages/elm/core/latest/String) ou [`Maybe`](https://package.elm-lang.org/packages/elm/core/latest/Maybe). Il faudrait pouvoir partager des données entre pages.
+3. **Code redondant.** La page d'accueil et la documentation partagent de nombreuses fonctions, comme `Html.text` et `Html.div`. Est-ce qu'on pourrait partager ce code entre pages ?
```

Comment:
Nit entre *les* pages

Reply:
Merci pour les retours, que j'ai tous intégrés, sauf celui-là ! Je trouve que c'est mieux avec "partager ce code entre pages", comme on prendrait un verre entre ami.e.s. Mais je suis prêt à renoncer si Vox Populi préfère ta version ;)

Reply:
Alors vraiment YOLO là-dessus, ship it 🚀